radiation/python/radlw/taumol_gt4py.py

```diff
@@ -0,0 +1,4957 @@
+        elif var in integervars or var in fltvars:
+            indict[var] = np.tile(tmp[None, None, :], (npts, 1, 1))
+        else:
+            indict[var] = tmp[0]
+
+indict_gt4py = dict()
+
+for var in invars:
+    if var == "NGB":
+        indict_gt4py[var] = create_storage_from_array(
+            indict[var],
+            backend,
+            shape_2D,
+            (DTYPE_INT, (ngptlw,)),
+            default_origin=(0, 0),
+        )
+    elif var == "colamt":
+        indict_gt4py[var] = create_storage_from_array(
+            indict[var], backend, shape_nlay, type_maxgas
+        )
+    elif var == "wx":
+        indict_gt4py[var] = create_storage_from_array(
+            indict[var], backend, shape_nlay, type_maxxsec
+        )
+    elif var == "tauaer":
+        indict_gt4py[var] = create_storage_from_array(
+            indict[var], backend, shape_nlay, type_nbands
+        )
+    elif var == "rfrate":
+        indict_gt4py[var] = create_storage_from_array(
+            indict[var], backend, shape_nlay, (DTYPE_FLT, (nrates, 2))
+        )
+    elif var in integervars:
+        indict_gt4py[var] = create_storage_from_array(
+            indict[var], backend, shape_nlay, DTYPE_INT
+        )
+    elif var in fltvars:
+        indict_gt4py[var] = create_storage_from_array(
+            indict[var], backend, shape_nlay, DTYPE_FLT
+        )
+    elif var == "fracs" or var == "tautot":
+        indict_gt4py[var] = create_storage_zeros(backend, shape_nlay, type_ngptlw)
+    else:
+        indict_gt4py[var] = indict[var]
+
+locdict_gt4py = dict()
+
+locvars_int = [
+    "ib",
+    "ind0",
+    "ind0p",
+    "ind1",
+    "ind1p",
+    "inds",
+    "indsp",
+    "indf",
+    "indfp",
+    "indm",
+    "indmp",
+    "js",
+    "js1",
+    "jmn2o",
+    "jmn2op",
+    "jpl",
+    "jplp",
+    "id000",
+    "id010",
+    "id100",
+    "id110",
+    "id200",
+    "id210",
+    "id001",
+    "id011",
+    "id101",
+    "id111",
+    "id201",
+    "id211",
+    "jmo3",
+    "jmo3p",
+    "jmco2",
+    "jmco2p",
+    "jmco",
+    "jmcop",
+    "jmn2",
+    "jmn2p",
+]
+locvars_flt = [
+    "taug",
+    "pp",
+    "corradj",
+    "scalen2",
+    "tauself",
+    "taufor",
+    "taun2",
+    "fpl",
+    "speccomb",
+    "speccomb1",
+    "fac001",
+    "fac101",
+    "fac201",
+    "fac011",
+    "fac111",
+    "fac211",
+    "fac000",
+    "fac100",
+    "fac200",
+    "fac010",
+    "fac110",
+    "fac210",
+]
+
+for var in locvars_int:
+    if var == "ib":
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_2D, DTYPE_INT)
+    else:
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlay, DTYPE_INT)
+
+for var in locvars_flt:
+    if var == "taug":
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlay, type_ngptlw)
+    else:
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlay, DTYPE_FLT)
+
+
+def loadlookupdata(name):
+    """
+    Load lookup table data for the given subroutine
```

Comment:
Might want to add to this a sentence about how if/when gt4py adds support for lookup tables, this should be reconsidered and what would change. It would still need to be loaded, but might be in a dict instead of a storage.

Reply:
Will do